robotframework-ls/src/robotframework_ls/impl/semantic_tokens.py

```diff
@@ -356,17 +443,18 @@ def semantic_tokens_full(context: ICompletionContext):
         ast = context.doc.get_ast()
     except:
         return []
+    return semantic_tokens_full_from_ast(ast, context.monitor)
```

Comment:
This change was recently done and was undone in this commit (so, the changes which add back `semantic_tokens_full_from_ast` should be reverted).

Reply:
Fixed